dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -197,7 +197,7 @@ def authenticate
         PartialRegistration.persist_attributes(session, user)
         session[:user_return_to] = destination_url
         if DCDO.get('lti_account_linking_enabled', false)
-          redirect_to lti_v1_account_linking_landing_path and return
+          redirect_to lti_v1_account_linking_landing_path, lti_provider: integration[:platform_name] and return
```

Comment:
```suggestion
          redirect_to lti_v1_account_linking_landing_path lti_provider: integration[:platform_name] and return
```
Locally this is only working for me without the comma